app/cdap/components/GridTable/components/NestedMenu/index.tsx

```diff
@@ -72,29 +74,37 @@ export default function({
         }}
         className={classes.root}
       >
-        {menuOptions?.map((eachOption, index) => (
-          <MenuItemComponent
-            item={eachOption}
-            columnType={columnType.toLowerCase()}
-            index={index}
-            onMenuClick={handleMenuClick}
-          />
-        ))}
-        <MenuComponent
-          anchorEl={anchorEl2}
-          columnType={columnType.toLowerCase()}
-          menuOptions={nestedOptions}
-          setAnchorEl={setAnchorEl2}
-          submitOption={(e, item) => {
-            e.preventDefault();
-            e.stopPropagation();
-            setAnchorEl(null);
-            handleMenuOpenClose(title);
-            setAnchorEl2(null);
-            submitMenuOption(item.value, item.supported_dataType);
-          }}
-        />
+        {menuOptions?.map((eachOption, index) => {
```

Comment:
change index to optionIndex, mandated rule for our project

if i have an array say, menuOptions
then each item inside would be eachOption
and each index would be optionIndex

e.g. ongoingExplorationsCards.map(eachCard, cardIndex)

Reply:
Done